src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,74 @@
  */
 
 public class Plane {
+    private String name;
+    private int maxSpeed;
+    private int capacityPassagers;
+    private int capacityFuel;
+    private int weight;
 
+    public Plane(PlaneBuilder builder) {
```

Comment:
let's make it private, we should have access for this constructor only within this class